### PR TITLE
MAP-1358 test refactoring

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

**Review by marcusaleman — COMMENTED** [review]
+1
I have checked just to familiarize with Prison-Register and Local Container in this case.

File: build.gradle.kts [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.15.3"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.1"
   kotlin("plugin.spring") version "1.9.22"
   kotlin("plugin.jpa") version "1.9.22"
```
Comment (danbenton-mojdt) [inline build.gradle.kts:6]: Is 2.0.0 compatible?

**Review by Mjwillis — APPROVED** [review]
LGTM

**Review by jnioche-jd — APPROVED** [review]
LGTM

**Pull request merged**